README.md

```diff
@@ -50,6 +52,14 @@ __Installation on Windows systems is as of current not possible. Please refer to
 the [Windows subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10). 
 You may also use the Singularity image of CAPICE found [here](https://download.molgeniscloud.org/downloads/vip/images/).__
 
+### SpliceAI
+
```

Comment:
remove newline

Reply:
Each of the headers within the README has a newline between the header and the content. For consistency reasons, remove all newlines after headers or keep this newline?